clase 1 19.4/src/main/java/services/GamePlay.java

```diff
@@ -0,0 +1,288 @@
+package services;
+
+import interfaces.IBoardBuilder;
+import interfaces.IGamePlay;
+import scanner.IScanner;
+import scanner.ScannerClass;
+import scanner.impl.MenuUserDecisionImpl;
+import java.util.*;
+
+public class GamePlay implements IGamePlay {
+
+    private IBoardBuilder boardBuilder;
+
+    public GamePlay(IBoardBuilder boardBuilder) {
+        this.boardBuilder = boardBuilder;
+    }
+
+    // Ejecutar 8vo
+    @Override
+    public Map<Integer, Integer> findZeroPosition(){
```

Comment:
Looks like all find methods are similar, so, you can create a flag or enum or command to execute an action